packages/@angular/cli/models/webpack-configs/typescript.ts

```diff
@@ -7,6 +7,8 @@ import {
   AngularCompilerPluginOptions,
   PLATFORM
 } from '@ngtools/webpack';
+// @ignoreDep typescript - used only for type information
+import * as ts from 'typescript';
```

Comment:
The usage below does not look like using only for type information. Looks as much value as it can get :)

    options.compilerOptions.module = ts.ModuleKind.CommonJS;